src/sentry/rules/conditions/event_frequency.py

```diff
@@ -261,15 +261,24 @@ def disable_consistent_snuba_mode(
         return option_override_cm
 
     def get_comparison_start_end(
-        self, interval: timedelta, duration: timedelta
+        self, end: datetime, duration: timedelta, comparison_interval: timedelta | None = None
```

Comment:
can we call `comparison_interval` `comparison_offset` instead?

The verbage is a bit confusing